### ISSUE TITLE
Front matter tags issue

### ISSUE THREAD

**Issue by joellti** [Gorkycreator/obsidian-quick-tagger#6]
With the most recent Obsidian desktop versions, tags can be added by dragging from tag view into the property section. The tags added this way will be preceded by # and then wrapped by double quotes. For example, tag `test` will become `"#test"` in the property section. I notice that tags in such format is not currently supported by this plugin. Is there a plan to do so in the future? Thanks.

**Comment by Gorkycreator** [comment]
Thanks for bringing this to my attention! I was able to repeat this behavior on my vault. As I was testing, though, I found that the drag-and-drop behavior is specifically provided by [tag-wrangler](https://github.com/pjeby/tag-wrangler).

Since Obsidian recognizes it, I'll add in support to read this formatting, but quick-tagger will conform processed notes to omit any pre-existing hashtags. I feel that's more in-line with Obsidian's default behavior.
